game/theater/controlpoint.py

```diff
@@ -69,6 +70,7 @@ class LocationType(Enum):
     MissileSite = "missile site"
     OffshoreStrikeTarget = "offshore strike target"
     Sam = "SAM"
+    Scenery = "scenery"
```

Comment:
Am skipping this file and `start_generator` because I think there's a larger design discussion that needs to be had first. Will follow up on the main PR thread.